### ISSUE TITLE
Allow Parker to load cross-domain content from Exhibits

### ISSUE THREAD

**Issue by blalbrit** [sul-dlss/exhibits#1077]
**Problem:** Embed widget in parker.stanford.edu does not load content (though does load text):
![screenshot 2018-02-02 09 59 34](https://user-images.githubusercontent.com/796990/35747644-d2e01424-07ff-11e8-82ae-6c8b6b55fe69.png)
![screenshot 2018-02-02 09 59 22](https://user-images.githubusercontent.com/796990/35747645-d2f38c02-07ff-11e8-8531-f86acd6b114c.png)
* from https://parker.stanford.edu/parker/feature/worlds-real-and-imagined

@jkeck  investigated briefly and determined that the cross-domain content load is being blocked by CORS. Confirmed by checking the same page through the non-aliased URL: https://exhibits.stanford.edu/parker/feature/worlds-real-and-imagined

**Desired outcome:** unblocking the content in the parker.stanford.edu view

(also leads to a follow-on question about hiding the UV contents panel in the widget by default - but that likely needs its own ticket).

**Comment by blalbrit** [comment]
Corpus team is requesting now because they cannot build effective feature pages without it.

**Comment by cbeer** [comment]
I wonder if the load-balancer that's handling the parker.stanford.edu alias isn't sending enough information to the application.

**Comment by jkeck** [comment]
When I curl the exhibit's oembed endpoint I'm not getting any `Access-Control-Allow-Origin` header.

**Comment by cbeer** [comment]
I'm really asking -- why it is cross-origin in the first place? `parker.stanford.edu` ought to use `parker.stanford.edu/oembed`.

**Comment by jkeck** [comment]
Maybe this is another engine routing issue?  (happy to say it's a load balancer issue and let ops investigate though)
